shared/devices/device-page/index.js

```diff
@@ -0,0 +1,77 @@
+import React, {Component} from 'react'
+import {bindActionCreators} from 'redux'
+import {connect} from 'react-redux'
+import moment from 'moment'
+
+import Render from './index.render'
+import RemoveDevice from '../device-revoke'
+
+import * as devicesActions from '../../actions/devices'
+import {routeAppend} from '../../actions/router'
+
+export default class DevicePage extends Component {
+  static parseRoute (currentPath) {
+    return {
+      componentAtTop: {
+        title: 'Device page',
```

Comment:
When we click on a device in the main device list, the device we chose needs to make it over to the existing-device page.  Decided to pass it through in the redux payload instead of putting a copy of the selected device into the store as devices: currentDevice: or something.